web/models/types/entity_type.ex

```diff
@@ -0,0 +1,27 @@
+defmodule CoursePlanner.Types.EntityType do
+  @moduledoc """
+    This module introduces a custom type for Etco for checking status in the model
+  """
+  @behaviour Ecto.Type
+  def type, do: :string
```

Comment:
Maybe you're right. I can't find a definite response on what I've found. Ultimately the value is a string, so not sure.
@ghatighorias does it work like this?